src/create.rs

```diff
@@ -57,7 +57,8 @@ impl Party0 {
         let TX_f = FundingTransaction::new(
             (x_self.public(), tid_self.clone()),
             (X_other, tid_other.clone()),
-        );
+        )
+        .expect(todo!("Should we use anyhow everywhere?"));
```

Comment:
I'm happy to use `anyhow`. We should still model most errors as structs though.

Reply:
I am not sure there is much point using `anyhow` if we model most errors as `enum`s. Did you mean `struct` or `enum`? I would be keen to use `thiserror` instead to make printing and conversion easy.